src/library/scala/collection/SortedSet.scala

```diff
@@ -42,7 +42,7 @@ trait SortedSet[A] extends Set[A] with SortedSetOps[A, SortedSet, SortedSet[A]]
 }
 
 trait SortedSetOps[A, +CC[X] <: SortedSet[X], +C <: SortedSetOps[A, CC, C]]
-  extends SetOps[A, Set, C]
+  extends SetOps[A, CC, C]
```

Comment:
This is interesting. It means that the `map` implementation inherited from `SetOps` will return a `SortedSet` although it has no way to get the implicit `Ordering[B]` (where `B` is the element type of the resulting collection). I think this should not compile.

Reply:
Updated with an alternative hopefully more correct fix.